src/google-apis/auth-client.ts

```diff
@@ -15,21 +15,16 @@
  */
 /*jshint unused:false*/
 
-export interface Service {
-  new(config: any, options: any): Service;
-  request: Function;
-}
-
-export interface Common {
-  Service: Service;
-}
-
-const common: Common = require('@google-cloud/common');
+import * as common from '@google-cloud/common';
 const pkg = require('../../../package.json');
 import * as is from 'is';
 var isFunction = is.fn;
 var isString = is.string;
 
+import {Configuration} from '../configuration';
+import {ErrorMessage} from '../classes/error-message';
+import * as http from 'http';
+
 /* @const {Array<String>} list of scopes needed to work with the errors api. */
 var SCOPES = ['https://www.googleapis.com/auth/cloud-platform'];
```

Comment:
Why `var`?  Should we do a run through and make sure we're using `const` and `let` everywhere?

Reply:
There will be an PR coming up that will be specifically for updating the `vars` to `const` and `let`.